neon_utils/logger.py

```diff
@@ -27,7 +27,10 @@
 # SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 from ovos_utils.log import LOG
+from neon_utils.log_aggregators import init_log_aggregators
```

Comment:
The `logger` module will be deprecated, so I'd drop these changes and expect each module to call `init_sentry`

Reply:
> The `logger` module will be deprecated, so I'd drop these changes and expect each module to call `init_sentry`

Per our previous discussion, I think it is worth to keep it here for now